src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.spec.tsx

```diff
@@ -232,14 +233,15 @@ describe('CourseProductItem', () => {
     const { product } = relation;
     const order = OrderFactory({
       product: product.id,
-      course: '00000',
+      course: CourseLightFactory({ code: '00000' }).one(),
       target_courses: product.target_courses,
     }).one();
 
     fetchMock.get(`https://joanie.test/api/v1.0/courses/00000/products/${product.id}/`, relation);
     const orderQueryParameters = {
       product: order.product,
-      course: order.course,
+      // TODO (rlecellier): remove this when order.course is mandatory
```

Comment:
order.course and order.enrollment are optional.
I would like to make them mandatory:  `course: CourseLight` instead of `course?: CourseLight`.
As it could imply more code changes, i thought not to do it here but in a further PR